src/SupervisionSuccess/model/transformData.js

```diff
@@ -0,0 +1,73 @@
+import calcRevocations from "./calcRevocations";
+import calcSavings from "./calcSavings";
+
+/* Number of months to add to the end of chart (chart need to be continued),
+This months should not participate in savings and prisonPopulationDiff calculations
+*/
+const ADDED_MONTHS = 5;
```

Comment:
Is this just to ensure the line extends to the edge of the allotted area as shown in the mocks? If so, I'd want to know if that number is correct for all screen widths.

Reply:
Yes, this is number of months to add to the end of chart to provide its overflowing to the edge of block.
This number is OK because we can control overflow value with media queries.